config/webpack.config.web2.js

```diff
@@ -0,0 +1,255 @@
+// @flow
+const express = require("express");
+/*::
+import type {
+  $Application as ExpressApp,
+  $Response as ExpressResponse,
+} from "express";
+*/
```

Comment:
nit: remove comment

Reply:
This is actually needed for flow checking -- this is a webpack config file, so it doesn't get run thru babel (since we integrate babel via webpack) so we needed to add the flow comments in comment-style syntax which is also valid javascript